plugins/node/opentelemetry-instrumentation-express/examples/package.json

```diff
@@ -21,24 +21,24 @@
     "tracing"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=8.12.0"
   },
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.0.4",
-    "@opentelemetry/exporter-jaeger": "^1.1.1",
-    "@opentelemetry/exporter-zipkin": "^1.1.1",
-    "@opentelemetry/instrumentation": "^0.27.0",
+    "@opentelemetry/api": "^1.0.0",
+    "@opentelemetry/exporter-jaeger": "^1.0.0",
+    "@opentelemetry/exporter-zipkin": "^1.0.0",
+    "@opentelemetry/instrumentation": "^0.28.0",
     "@opentelemetry/instrumentation-express": "^0.28.0",
-    "@opentelemetry/instrumentation-http": "^0.27.0",
-    "@opentelemetry/resources": "^1.1.1",
-    "@opentelemetry/sdk-trace-base": "^1.1.1",
-    "@opentelemetry/sdk-trace-node": "^1.1.1",
-    "@opentelemetry/semantic-conventions": "^1.1.1",
```

Comment:
Here, because in other locations, sdk packages are not updated.

Reply:
done, mostly cleanup oversights from previous PRs or merge races.